sdk/tests/deqp/functional/gles3/es3fFboInvalidateTests.js

```diff
@@ -804,7 +804,7 @@ es3fFboInvalidateTests.InvalidateSubFboUnbindReadCase.prototype.render = functio
     var attachments = getFBODiscardAttachments(this.m_invalidateBuffers);
     // Create fbo.
     var transferFmt = gluTextureUtil.getTransferFormat(colorFmt);
-    var gradShader = new es3fFboTestUtil.GradientShader(gluShaderUtil.DataType.FLOAT_VEC4);
+    var gradShader = new es3fFboTestUtil.GradientShader(es3fFboTestUtil.getFragmentOutputType(colorFmt));
```

Comment:
C++ code:
https://android.googlesource.com/platform/external/deqp/+/master/modules/gles3/functional/es3fFboInvalidateTests.cpp#947
If the type of the input color is integer, the fragment shader here will not run correctly.